docker/Dockerfile.package-cpu

```diff
@@ -1,6 +1,6 @@
 # Docker image: tlcpack/package-cpu
 
-FROM quay.io/pypa/manylinux2014_x86_64:2022-02-13-594988e
+FROM pytorch/manylinux-cpu
```

Comment:
I don't love that it's not version pinned but looking at the Hub entry for it it seems like they get rid of all but the latest version. We can merge this now but we should probably look into recreating the `pytorch/manylinux-cpu` image ourselves so we can have a stable base